src/ImageProcessor.Web/Helpers/UrlDecoder.cs

```diff
@@ -0,0 +1,42 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="RemoteFile.cs" company="James Jackson-South">
+//   Copyright (c) James Jackson-South.
+//   Licensed under the Apache License, Version 2.0.
+// </copyright>
+// <summary>
+//   Encapsulates methods used to download files from a website address.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace ImageProcessor.Web.Helpers
+{
+    using System;
+
+    /// <summary>
+    /// Helper class for decoding passed urls
+    /// </summary>
+    public class UrlDecoder
+    {
+        /// <summary>
+        /// A new instance of the <see cref="UrlDecoder"/> class.
+        /// with lazy initialization.
+        /// </summary>
+        private static readonly Lazy<UrlDecoder> Lazy = new Lazy<UrlDecoder>(() => new UrlDecoder());
```

Comment:
This doesn't need to have the overhead of Lazy<T> a simple static class would suffice since the method is threadsafe.